src/assets/contributors-map.geojson

```diff
@@ -239,6 +239,20 @@
           -112.0740,
           33.4484
         ]
+      },
```

Comment:
I think there's a missing `}`.  Can you double check?

Reply:
thanks for catching! I pushed the fix. 

Also, minor update to the docs referencing this filename as contributors-map.json (instead of geojson) https://github.com/OpenBeta/docs.openbeta.io/pull/47